packages/scaffolder/src/features/store.ts

```diff
@@ -273,14 +282,14 @@ export class FeatureStore {
       }
     }
 
-    // TODO: Enable this in a follow up PR.
-    /* eslint-disable */
     // Check if we're developing on the scaffolder locally.
-    // if (__dirname.includes('packages/scaffolder') && fs.existsSync(path.join(__dirname, '..', '..', '..', '..', 'node_modules'))) {
-      // console.log('adding', path.join(__dirname, '..', '..', '..', '..', 'node_modules'));
-      // paths.push(path.join(__dirname, '..', '..', '..', '..', 'node_modules'));
-    // }
-    /* eslint-enable */
+    if (__dirname.includes('packages/scaffolder')) {
+      const projectRoot = path.join(__dirname, '..', '..');
+
+      if (fs.existsSync(path.join(projectRoot, '..', 'scaffolder-features'))) {
+        paths.push(path.join(projectRoot, '..', 'scaffolder-features'));
+      }
+    }
```

Comment:
In the build-tool package, `cwd()` was used to get the project root as the tool is meant to be used on the project root. My question is, will this always be true?
```ts
const projectRoot = path.join(__dirname, '..', '..');
```